### DISCUSSION TITLE
no module named auto-py-to-exe

### DISCUSSION THREAD

**Discussion by heysailboat** [brentvollebregt/auto-py-to-exe#480] [Q&A]
After cloning, `cd`-ing into the directory, and running, still no luck. Deleted the folder and tried using `pip`. Still no. Tells me the module isn't found. Can someone help?

**Comment by brentvollebregt** [comment]
What exactly are you running? 

Also what are you doing before trying to run the tool? To be clear, you can't just download, cd and run `auto-py-to-exe` - you need to install it first.

**Reply by heysailboat** [reply]
Sorry, I found a fix! Just needed to clone using HTML, cd into the directory and run `python3 run.py` after installing dependencies.
